docs/podman.rst

```diff
@@ -0,0 +1,51 @@
+.. _podman:
+
+Running Tutor with Podman
+-------------------------
+
+You have the option of running Tutor with `Podman <https://podman.io/>`__, instead of the native Docker tools. This has some practical advantages: it does not require a running Docker daemon, and it enables you to run and build Docker images without depending on any system component running ``root``. As such, it is particularly useful for building Tutor images from CI pipelines.
+
+The ``podman`` CLI aims to be fully compatible with the ``docker`` CLI, and ``podman-compose`` is a fully-compatible alias of ``docker-compose``. This means that you can use both together with Tutor, without making any changes to Tutor itself.
+
+.. warning::
+   You should not attempt to run Tutor with Podman on a system that already has native ``docker`` and ``docker-compose`` installed. If you want to switch to ``podman`` and ``podman-compose`` using the aliases described here, uninstall the native Docker packages first. 
+
+
+Enabling Podman
+^^^^^^^^^^^^^^^
+
+Podman is supported on a variety of development platforms, see the `installation instructions <https://podman.io/getting-started/installation>`_ for details.
```

Comment:
@regisb:

> Meanwhile, I suggest you provide less documentation on how to install Podman, and instead point  > to the relevant upstream documentation [page](https://podman.io/getting-started/installation).

Yeah it does exactly that, here. It then goes on to explain what *else* is necessary to make *Tutor* play nicely with Podman, which I do consider relevant, and which isn't discussed in the upstream docs.